pyaccel/elements.py

```diff
@@ -271,6 +283,17 @@ def y_nrpts(self):
         """."""
         return self._kicktable.y_nrpts
 
+    @property
+    def status(self):
+        """Return last object status."""
+        return self._status
+
+    def get_kicks(self, rx, ry):
+        """Return (kickh, vkick) at (rx,ry)."""
+        idx = self.kicktable_idx
+        self._status, hkick, vkick = _trackcpp.kicktable_getkicks_wrapper(idx, rx, ry)
```

Comment:
ahh, ok ok, I think now I understand the use of `typemap.i` and the `%apply` syntax in trackcpp PR! Very useful!

Reply:
yes, it solves the problem of mapping c++ functions with changing input arguments by transforming them in output arguments.

Reply:
I remember when I originally wrote pyaccel.Kicktable that I did not implement 'get_kicks' at that time because I did not know how to do it! I just learned about this input/output conversion yesterday. it maybe be handy to have 'get_kicks' available in python.